interfaces/IRepository.php

```diff
@@ -3,8 +3,24 @@
 
 interface IRepository
 {
+    /**
+     * Funkcja wyszukuje element w bazie Repozytorium i go zwraca
+     * @param $id - id szukanego elementu
+     */
     function find($id);
+    /**
+     * Funkcja wyszukuje wszystkie elementy w bazie Repozytorium i je zwraca
+     *
+     */
     function findAll();
+    /**
+     * Funkcja usuwa element z bazy repozytorium
```

Comment:
Słowo repozytorium jest tu niepotrzebne. Przydało by się za to powiedzieć, że jest to pojedynczy element o zadanym id ;)

Reply:
Analogicznie dla pozostałych funkcji